fastn-core/src/commands/serve.rs

```diff
@@ -186,6 +186,10 @@ pub async fn serve(
         return Ok(clear_sid2(&req));
     }
 
+    if let Some(endpoint_response) = handle_migrations(config, &req).await {
+        return endpoint_response;
```

Comment:
Why would migration return a reponse? Also migration can error, so its return type must be `Result`.